### PR TITLE
Class1 data

### PR DESCRIPTION
**Pull request by aguevara1**
Class 1 Project

### CODE REVIEW

**Comment by aguevara1** [comment]
Please look at the "corrected data type function file"

File: 01week/datatypes.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+}
+addTwoNumbers(57, 388);
+
+
+function twoThingsTrue(thing1, thing2) {
+  const message = 'both true';
+  if (thing1 && thing2) {
+    return message;
+  } else {
+    return false;
```
Comment (reneemeyer) [inline 01week/datatypes.js:56]: else return is unnecessary, the default is to return null

File: 01week/datatypes.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+twoThingsTrue(true, 66);
+
+
+function oneOfTwo(thing3, thing4) {
+  const message1 = 'One of two are true';
+  if (thing3 || thing4) {
+    return message1;
+  } else {
+    return false;
```
Comment (reneemeyer) [inline 01week/datatypes.js:68]: else return is unnecessary, the default is to return null

File: 01week/datatypes.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+oneOfTwo(8, 9);
+
+
+function bothNotTrue(thing5, thing6) {
+  const message3 = 'both not true';
+  if (!thing5 && !thing6) {
+    return message3;
+  } else {
```
Comment (reneemeyer) [inline 01week/datatypes.js:79]: else return is unnecessary, the default is to return null

**Review by reneemeyer — APPROVED** [review]
Nice job on the PR and the assignment! Go ahead and merge the PR and delete the branch.

**Pull request merged**